mapusaurus/api/tests.py

```diff
@@ -32,7 +32,7 @@ class ViewsTests(TestCase):
 
     def test_api_all_user_errors(self):
```

Comment:
Might want to rename this test. Previously it was verifying a HttpResponseBadRequest was being generated; now it's verifying that such a response *isn't* generated (?) It'd probably be worth while to test that these user inputs give expected outputs rather than just not raising an error.